test/manifests/invalid.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
nit: you could explain why this manifest is invalid via the filename (I actually don't know why this is invalid)